### ISSUE TITLE
Change TextStyle Issue.

### ISSUE THREAD

**Issue by DipakSkywave** [RyuuKenshi/flutter_collapsible_sidebar#6]
@RyuuKenshi How to change TextStyle in Menu?

Code : 

CollapsibleSidebar(
            items: _items,
            avatarImg:
                NetworkImage('https://www.w3schools.com/howto/img_avatar.png'),
            title: 'Test',
            iconSize: 25.0,
            textSize: 15.0,
            minWidth: 67.0,
            height: double.infinity,
            duration: Duration(milliseconds: 500),
            curve: Curves.fastLinearToSlowEaseIn,
            toggleButtonIcon: Icons.arrow_forward_ios,
            screenPadding: 4,
            borderRadius: 15,
            body: _body(size, context),
          ),

**Comment by RyuuKenshi** [comment]
Hi @DipakSkywave,

Thank you very much for pointing out this issue. I have fixed it and updated the package to v1.0.7+2.
The new properties I have added to `CollapsibleSidebar()` are:
```
textStyle: //for collapsible tiles text
titleStyle: //for sidebar title text
toggleTitleText: //for the toggle button text
```
using these you can use custom TextStyle in the Sidebar.
You can check out the other improvements at [https://pub.dev/packages/collapsible_sidebar](url)

Hope this update will help you.

**Comment by DipakSkywave** [comment]
> Hi @DipakSkywave,
> 
> Thank you very much for pointing out this issue. I have fixed it and updated the package to v1.0.7+2.
> The new properties I have added to `CollapsibleSidebar()` are:
> 
> ```
> textStyle: //for collapsible tiles text
> titleStyle: //for sidebar title text
> toggleTitleText: //for the toggle button text
> ```
> 
> using these you can use custom TextStyle in the Sidebar.
> You can check out the other improvements at [https://pub.dev/packages/collapsible_sidebar](url)
> 
> Hope this update will help you.

It's Working in v1.0.7+2.

Thanks @RyuuKenshi

**Comment by RyuuKenshi** [comment]
You are welcome @DipakSkywave.
